lib/assets/sql_cell/main.js

```diff
@@ -6,6 +6,10 @@ export function init(ctx, payload) {
   ctx.root.innerHTML = `
     <div class="app">
       <div>
+        <div id="conn-info-box" class="conn-box">
```

Comment:
Since we use this styling in other places, let's keep consistent names :)

```suggestion
        <div id="conn-info-box" class="info-box">
```

Reply:
info-box conflicts with the `#help-box` that already uses `info-box` class and uses pre-wrap.
Or we could use a prefix with element ID like `#conn-info-box.info-box`. WDYT?